src/helpers/UniV3PoolBalanceCalculator.test.ts

```diff
@@ -3,76 +3,87 @@ import { BigNumber } from "ethers";
+    3001, // fee
+    -332580, // tickLower
+    -331800, // tickUpper
+    BigNumber.from(89923569515171), // liquidity
   ];
+  public slot01 = [
+    BigNumber.from("1770416090971842202582039427480271301"), // sqrtPriceX96
+    0,
+    0,
+    0,
+    0,
+    0,
+    true,
+  ];
+
+  public slot02 = [
+    BigNumber.from("4295128740"), // sqrtPriceX96
+    0,
+    0,
+    0,
+    0,
+    0,
+    true,
+  ];
+
   public pool1Address = "pool1";
   public pool2Address = "pool2";
-  public pool1TotalLiqudity = BigNumber.from(100);
-  public pool2TotalLiqudity = BigNumber.from(100);
-  public pool1Token0Balance = BigNumber.from(1000);
-  public pool1Token1Balance = BigNumber.from(1000);
-  public pool2Token0Balance = BigNumber.from(1000);
-  public pool2Token1Balance = BigNumber.from(1000);
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  public async getPositionTokenIds(address: string): Promise<number[]> {
+  public async getPositionTokenIds(address: string): Promise<BigNumber[]> {
     return this.positionTokenIds;
   }
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public async getPosition(tokenId: number): Promise<any> {
-    if (tokenId === 2630) return this.position1;
-    if (tokenId === 2428) return this.position2;
+    if (tokenId === 3115) return this.position1;
+    if (tokenId === 3113) return this.position2;
   }
 
   public async getPoolAddress(
     token0: string,
     token1: string,
     fee: number,
   ): Promise<string> {
-    if (fee == 500) return this.pool1Address;
-    if (fee == 700) return this.pool2Address;
+    if (fee == 3000) return this.pool1Address;
+    if (fee == 3001) return this.pool2Address;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  public async getPoolBalance(
-    poolAddress: string,
-    _asset0: string,
-    _asset1: string,
-  ): Promise<BigNumber[]> {
-    if (poolAddress === this.pool1Address)
-      return [this.pool1Token0Balance, this.pool1Token1Balance];
-    if (poolAddress === this.pool2Address)
-      return [this.pool2Token0Balance, this.pool2Token1Balance];
+  public async getSlot0(poolAddress: string): Promise<any> {
+    if (poolAddress === this.pool1Address) return this.slot01;
+    if (poolAddress === this.pool2Address) return this.slot02;
   }
 
-  public async getTotalLiquidityForPool(
-    poolAddress: string,
-  ): Promise<BigNumber> {
-    if (poolAddress === this.pool1Address) return this.pool1TotalLiqudity;
-    if (poolAddress === this.pool2Address) return this.pool2TotalLiqudity;
-  }
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public async getERC20Decimals(tokenAddress: string): Promise<number> {
+    if (tokenAddress === "0x765DE816845861e75A25fCA122bb6898B8B1282a")
+      return 18;
+    if (tokenAddress === "0x1a35EE4640b0A3B87705B0A4B45D227Ba60Ca2ad") return 8;
+    if (tokenAddress === "0xd71Ffd0940c920786eC4DbB5A12306669b5b81EF")
+      return 18;
```

Comment:
maybe assign the addresses to variables for better readability? i.e. `cUSD = 0x..`

Reply:
sounds good 👍